plugins/chat/config/locales/server.en.yml

```diff
@@ -240,18 +240,20 @@ en:
         other: "You have new chat messages"
       from: "%{site_name}"
       subject:
-        private_message: "[%{email_prefix}] New message"
-        direct_message_from_1: "[%{email_prefix}] New message from %{username}"
-        direct_message_from_2: "[%{email_prefix}] New message from %{username1} and %{username2}"
-        direct_message_from_more:
-          one: "[%{email_prefix}] New message from %{username} and %{count} other"
-          other: "[%{email_prefix}] New message from %{username} and %{count} others"
-        chat_channel_1: "[%{email_prefix}] New message in %{channel}"
-        chat_channel_2: "[%{email_prefix}] New message in %{channel1} and %{channel2}"
-        chat_channel_more:
-          one: "[%{email_prefix}] New message in %{channel} and %{count} other"
-          other: "[%{email_prefix}] New message in %{channel} and %{count} others"
-        chat_channel_and_direct_message: "[%{email_prefix}] New message in %{channel} and from %{username}"
+        private_email:
```

Comment:
Renamed the translation strings to have better consistency.

Also used `%{site_name}` instead of `%{email_prefix}` since it seemed to be more popular. (They both have the same value).